src/EnergyPlus/PurchasedAirManager.cc

```diff
@@ -2900,6 +2901,8 @@ void CalcPurchAirLoads(EnergyPlusData &state,
         PurchAir(PurchAirNum).OALatOutput = 0.0;
         PurchAir(PurchAirNum).MixedAirTemp = Node(RecircNodeNum).Temp;
         PurchAir(PurchAirNum).MixedAirHumRat = Node(RecircNodeNum).HumRat;
+        PurchAir(PurchAirNum).SupplyTemp = Node(InNodeNum).Temp;
+        PurchAir(PurchAirNum).SupplyHumRat = Node(InNodeNum).HumRat;
```

Comment:
This all looks reasonable but I am unsure of the difference between setting the supply node condition to the inlet versus the mixed node when the unit is off. @mjwitte ? And when EMS is setting the conditions, regardless if there is flow, shouldn't the supply condition reflect what EMS is writing to the node? I guess at 0 mass flow you can't have a delta T, w or H so this does make sense this way.

Reply:
@rraustad The supplytemp and supplyhumrat struct variables were added recently so that new output variables could be added. Since these are the conditions at the InNode, then they better have the same value. When flow is zero, the InNode conditions are set to match the zone node. That was done long ago, probably to make sure there aren't any funny noise values in the zone air balance calcs.